src/gui/mzroll/mainwindow.cpp

```diff
@@ -2458,6 +2465,29 @@ int MainWindow::loadMetaCsvFile(string filename){
 
         }
 
+        qreal *redF, *greenF, *blueF, *alpha;
+        if (header.count("color") && header["color"] < N) {
+
+            string colorName = fields[header["color"]];
+            redF = new qreal(-1);
+            greenF = new qreal(-1);
+            blueF = new qreal(-1);
+            alpha = new qreal(-1);
+            QColor color;
+            
+            if(!colorName.empty()) {
+                
+                if (colorName[0] == '#') {
+                    color.setNamedColor(QString::fromStdString(colorName));
+                    color.getRgbF(redF, greenF, blueF, alpha);
+                } else if (rootxkcd_colors[colorName] != nullptr) { 
+                    string hexValue = rootxkcd_colors[colorName];
```

Comment:
If the name of a color does not exist, what does the library return? Quoting a warning from the library's README:
> In function `from_json`, use function `at()` to access the object values rather than `operator[]`. In case a key does not exist, `at` throws an exception that you can handle, whereas `operator[]` exhibits undefined behavior.

If by chance it gives undefined behavior or even an unhandled exception, that would be bad. I did try a color name that did not exist in our xkcd json file, but the software did not crash. Which is good. Still, I would like there to be a check for something like this.